app/views/contexts/summary/question.py

```diff
@@ -61,7 +60,7 @@ def _build_answers(
                 list_item_id=self.list_item_id,
                 return_to=return_to,
                 return_to_answer_id=answer_id if return_to else None,
-                _anchor=answer_id,
+                _anchor=question_schema.get("answers")[0]["id"],
```

Comment:
"answers" is mandatory
```suggestion
                _anchor=question_schema["answers"][0]["id"],
```